lib/b2b/passwords_email.ts

```diff
@@ -4,23 +4,53 @@
 // or your changes may be overwritten later!
 // !!!
 
-import {} from "../shared/method_options";
+import {
+  Authorization,
+  addAuthorizationHeaders,
+} from "../shared/method_options";
 import { fetchConfig } from "../shared";
 import { Member, Organization } from "./organizations";
 import { MemberSession } from "./sessions";
 import { MfaRequired } from "./mfa";
 import { request } from "../shared";
 
-export interface B2BPasswordsEmailDeleteRequest {
```

Comment:
@bgier-stytch what's this method that was deleted? Was this ever supported?

Reply:
@jennifer-stytch - @max-stytch says it was never supported